changelog/unreleased/kong/fix-ai-streaming-resp-chunk-ending.yml

```diff
@@ -0,0 +1,3 @@
+message: "**ai-proxy**: Fixed a bug where the last chunk didn't end wih '\n\n'."
```

Comment:
typo `with`, and I think that this message may be a little technical for user.
Also, The message should have the prefix `Fixed an issue where ...`